features/appmgt/org.wso2.carbon.appmgt.store.feature/src/main/resources/admin-dashboard/site/themes/default/templates/xacmlpolicy-task/js/xacmlpolicy.js

```diff
@@ -312,41 +313,40 @@ function savePolicyPartial() {
             contentType: 'application/json',
             dataType: 'json',
             success: function (response) {
-
-                var apps = "";
-                if (response.length != 0) {
-                    // construct and show the  the warning message with app names which use this partial before update
-                    for (var i = 0; i < response.length; i++) {
-                        var j = i + 1;
-                        apps = apps + j + ". " + response[i].appName + "\n";
-                    }
-
-                    var msg = "policy " + policyPartialName + " is used in following apps " +
-                        apps +
-                        "Are you sure you want to modify the policy " + policyPartialName + "?";
-
-                    var conf = confirm(msg);
-                    if (conf == true) {
-                        updateModifiedPolicyPartial(editedpolicyPartialId, policyPartialName, generatedRule, ruleCondition, ruleEffect, isSharedPartial, policyPartialDesc);
-                    }
-                }
-                else {
-                    updateModifiedPolicyPartial(editedpolicyPartialId, policyPartialName, generatedRule, ruleCondition, ruleEffect, isSharedPartial, policyPartialDesc);
-                }
-
-
+                SavePolicyPartialsOnSuccess(response, policyPartialName, generatedRule, ruleCondition, ruleEffect, isSharedPartial, policyPartialDesc);
             },
             error: function (response) {
+                Showalert('Error occurred while updating entitlement policy content', "alert-error", "statusError");
             }
         });
-
-
     }
 
     resetControls();
 
 }
 
+function SavePolicyPartialsOnSuccess(response, policyPartialName, generatedRule, ruleCondition, ruleEffect, isSharedPartial, policyPartialDesc) {
+    var apps = "";
```

Comment:
I'm trying to concatenate and build a string using this variable. To avoid the value 'undefined' I'm assigning valiable to "" in the beginning.